src/api/index.js

```diff
@@ -117,6 +121,15 @@ class Steem extends EventEmitter {
         return this.transport.send(api, data, cb);
     }
 
+    call(method, params, callback) {
```

Comment:
I suggest that call() be renamed rpc()

Reply:
Just wondering why do we need this function? There is already `steem.api.send()` which give similar result and not specific to a transport.

Reply:
We are moving all services to jussi which will be a proxy for both steemd and our services and it uses jsonrpc differently (the rpc method isn't hard-coded to `call`) so I needed to separate it. We will also be adding request signing to this at some point.

My thinking is that we use `.call` for misc json rpc service calls for now and when we are ready to drop websocket entirely we route everything through the `.call` method and remove `.send`.

Re, naming I think `steem.api.call()` makes more sense than `steem.api.rpc()`